go.mod

```diff
@@ -218,4 +218,5 @@ replace (
 	github.com/gravitational/teleport/api => ./api
 	github.com/siddontang/go-mysql v1.1.0 => github.com/gravitational/go-mysql v1.1.1-teleport.1
 	github.com/sirupsen/logrus => github.com/gravitational/logrus v1.4.4-0.20210817004754-047e20245621
+	github.com/vulcand/predicate => github.com/gravitational/predicate v1.2.0
```

Comment:
Bump the dependency on `github.com/vulcand/predicate` to v1.2.0 as well, it's still at v1.1.0

Reply:
Huh, I guess we actually don't need to do that; I don't know what the best practices are, do we just leave whatever version was there at the time of replacement?

Reply:
Actually, considering that `v1.2.0` is identical for both, can't we just keep using `github.com/vulcand/predicate` for now and change the actual module name for `v1.3.0`, as it gets developed further? We only have 5 imports in here, we can just fix those as we swap the package

Reply:
@espadolini i did feel silly about both repos being same version, i did as you suggested, just upgrade vulcand